libraries/botframework-connector/tests/auth/userTokenClient.test.js

```diff
@@ -0,0 +1,27 @@
+const assert = require('assert');
+const { UserTokenClient } = require('../..');
+
+describe('UserTokenClient', function () {
+    describe('createTokenExchangeState', function () {
+        const paramsValidation = [
+            { paramName: 'appId', values: [null, null, null] },
+            { paramName: 'connectionName', values: ['appId', null, null] },
+            { paramName: 'activity', values: ['appId', 'connectionName', null] },
+        ];
+
+        paramsValidation.forEach((testConfig) => {
+            it(`should throw with null ${testConfig.paramName}`, function () {
+                // TODO: Unpack why botbuilder-stdlib.assert isn't workng with assert.throws().
```

Comment:
Previously, the code was a forEach (like [skillHandler.test.js](https://github.com/microsoft/botbuilder-js/blob/6bd641eb13dd4e09dc4bb4fac4b3155f3005922f/libraries/botbuilder/tests/skills/skillHandler.test.js#L99-L108)) and an `assert.throws` call. However, the _expected errors_ thrown from `botbuilder-stdlib` were not being inspected in `assert.throws` and just failing the test.